File: 1152.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String inputString = sc.nextLine();
        String trimmedString = inputString.trim();                                                                  //string.trim() trims front and lear of string
        int wordCounter = 1;
        if(trimmedString.length() == 0) wordCounter = 0;                                                            //trimmed string may have no word in string
        for(int i = 0; i < trimmedString.length(); i++)    if(trimmedString.charAt(i) == ' ')  wordCounter++;       //string.length() returns length of string
        System.out.println(wordCounter);
    }
}

File: 1152.py
inputString = input()

wordCounter = 1

for each in inputString:
    if each == ' ': wordCounter = wordCounter + 1
if inputString[0] == ' ':    wordCounter = wordCounter - 1
if inputString[len(inputString)-1] == ' ':    wordCounter = wordCounter - 1     #len func returns length of list
    
print(wordCounter)
